qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -5,6 +5,8 @@
 
 from shlex import split as shlex_split
 
+from tasks.ceph_manager import CephManager
```

Comment:
Commit says:

```
In other words, for CephFS tests, instead of writing
self.mds_cluster.mon_manager.run_cluster() in tests one can simply write
self.run_cluster_cmd().
```

Tests already invoke it that way, isn't it?

```git grep "run_cluster_cmd" qa/tasks/cephfs/test_*.py
qa/tasks/cephfs/test_admin.py:            self.run_cluster_cmd(f'osd pool create {p}')
qa/tasks/cephfs/test_admin.py:        self.run_cluster_cmd(f'fs new {fs_name} {pool_names[0]} {pool_names[1]} --fscid  {fscid} --force')
qa/tasks/cephfs/test_admin.py:            self.run_cluster_cmd(f'osd pool create {p}')
...
...
...
```

There are a couple of invocation that use `self.fs.mon_manager` - those can be switched to `self.run_cluster_cmd()`?

Reply:
> Commit says:

Which commit?

> Tests already invoke it that way, isn't it?

I was talking about run_cluster_cmd from CephManager. run_cluster_cmd in CephFSTestCase itself depends on variable `self.fs` being initialized. 
 
> There are a couple of invocation that use `self.fs.mon_manager` - those can be switched to `self.run_cluster_cmd()`?

Which ones?

Reply:
@rishabh-d-dave I'm a bit confused. Let me start over.

The idea is to avoid writing `self.mds_cluster.mon_manager.run_cluster_cmd`. AFAICS, there is just one invocation that does this:

```
git grep self.mds_cluster.mon_manager.run_cluster_cmd
qa/tasks/cephfs/cephfs_test_case.py:            self.mds_cluster.mon_manager.run_cluster_cmd(args="osd blocklist clear")
```

So, my question - is there a need to instantiate CephManager in cephfs_test_case.py to change the above (single) invocation to self.run_cluster_cmd()?

Reply:
It's not just about CephManager.run_cluster_cmd.(). There are many calls to raw_cluster_cmd_result() and many many calls to raw_cluster_cmd() which have been replaced in subsequent commits. Which is why this PR has so many changes.

I've updated the commit message, https://github.com/ceph/ceph/pull/50569/commits/6a1283cca9c00c2a938ea10224a9123f85d63c43, to include information about rest of the patches in this series hoping the intent will be clearer to the reader next time.

Reply:
ACK